### ISSUE TITLE
Bug with pycares 2.0.0

### ISSUE THREAD

**Issue by socketubs** [slimta/python-slimta#102]
As referenced [here](https://github.com/saghul/pycares/issues/30), pycares change its import paths in `2.0.0` which has been release recently and slimta failed to load it because slimta is pinning to `pycares>=1`.

```python
ERROR collecting test/test_slimta_relay_smtp_mx.py
test/test_slimta_relay_smtp_mx.py:3: in <module>
    from pycares.errno import ARES_ENOTFOUND, ARES_ENODATA
E   ImportError: No module named 'pycares.errno'

ERROR collecting test/test_slimta_util_dns.py
test/test_slimta_util_dns.py:5: in <module>
    import pycares.errno
E   ImportError: No module named 'pycares.errno'

ERROR collecting test/test_slimta_util_dnsbl.py
test/test_slimta_util_dnsbl.py:4: in <module>
    from pycares.errno import ARES_ENOTFOUND
E   ImportError: No module named 'pycares.errno'

383 passed, 3 error in 1.73 seconds
```

**Comment by socketubs** [comment]
Just pinging @saghul in case of any questions. :smile:

**Comment by socketubs** [comment]
Fixed in `2.0.1`.
